build/send-payload/Dockerfile

```diff
@@ -0,0 +1,9 @@
+## send-payload image ##
+FROM alpine
+COPY ./testdata/send-payload /send-payload
+WORKDIR /src
+RUN apk add curl openssl jq
+ENTRYPOINT /send-payload $0 $1 $2
```

Comment:
Does it work with `/send-payload "$@"` ? It would allow any number of options.

Reply:
I thought I tried and it didnt work, but as King Cromson stand "it just works" now :)

Reply:
Don’t be a schizoid man, it happens to everybody.

Reply:
I wanted to ask you to add quotes around `"$@"`, but when I take a look at https://github.com/imega-docker/jq/blob/master/Dockerfile, it reminds me that you don’t need to use `/send-payload $@`, simply use `/send-payload`, Docker will pass the arguments to the entrypoint script.

Reply:
interesting: it works only when ENTRYPOINT is fed array like parameters. Would not work with `ENTRYPOINT seq`